Tribler/Core/APIImplementation/LaunchManyCore.py

```diff
@@ -570,8 +570,9 @@ def sesscb_states_callback(self, states_list):
                 new_active_downloads.append(safename)
             elif state == DLSTATUS_STOPPED_ON_ERROR:
                 self._logger.error("Error during download: %s", repr(ds.get_error()))
-                self.downloads.get(tdef.get_infohash()).stop()
-                self.session.notifier.notify(NTFY_TORRENT, NTFY_ERROR, tdef.get_infohash(), repr(ds.get_error()))
+                if tdef.get_infohash() in self.downloads:
```

Comment:
Could you use `get_download(tdef.get_infohash())` for this? That method also makes sure the proper locks on the downloads structure are set.